src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -31,24 +27,23 @@ void reader_fileNotExist_notOk() {
 
     @Test
     void reader_fileIsEmpty_notOk() {
-        expected = new ArrayList<>();
-        expected = readerService.readFromFile("src/main/resources/input_file.csv");
+        List<String> actual = readerService.readFromFile("src/test/resources/input_file_test.csv");
         assertThrows(AssertionError.class,
-                () -> assertTrue(expected.isEmpty()),
+                () -> assertTrue(actual.isEmpty()),
                 "Should throw runtime exception.");
     }
```

Comment:
Let's refactor this test, as it was mentioned previously.

Reply:
it is better to test empty list in this way: `assertTrue(actual.isEmpty())`